documentation/modules/configuring/proc-creating-logging-filters.adoc

```diff
@@ -0,0 +1,167 @@
+// Module included in the following assemblies:
+//
+// assembly-logging-configuration.adoc
+
+[id='creating-logging-filters_{context}']
+= Adding logging filters to Operators
```

Comment:
This seems to be missing the main part ... how do I assign the different log levels to the different filters? (CC @sknot-rh )

Reply:
You don't. The markers are just filters. So you have the logging on (for example) DEBUG level you have a large number of logging messages. To reduce it, you focus on a single resource to be logged.

Reply:
I guess that is not clear really from the docs then. (it also makes this useless TBH because I do not think you can afford to not log the others plus it creates a confusion about what is going on in the cluster, but lets just make it clear in the docs)

Reply:
I've added the following:

> Logging filters are useful when you have a large number of logging messages.
> Suppose you set the log level for the logger as DEBUG (`rootLogger.level="DEBUG"`).
> Logging filters reduce the number of logs returned for the logger at that level, so you can focus on a specific resource.

Reply:
@PaulRMellor I think this is still not clear enough, I think it should say something like _When the filter is set, only log messages matching it will be logged._

Reply:
Thanks @scholzj . Added.

> Logging filters are useful when you have a large number of logging messages.
> Suppose you set the log level for the logger as DEBUG (`rootLogger.level="DEBUG"`).
> Logging filters reduce the number of logs returned for the logger at that level, so you can focus on a specific resource. When the filter is set, only log messages matching the filter are logged.